src/client/html/index.html

```diff
@@ -1,3 +1,4 @@
+<!DOCTYPE html>
```

Comment:
Technically required for proper HTML, and thus helps keep you out of quirks mode.